.idea/runConfigurations/Instrumentation_Tests.xml

```diff
@@ -0,0 +1,48 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Instrumentation Tests" type="AndroidTestRunConfigurationType" factoryName="Android Instrumented Tests">
```

Comment:
This is a file for the IDE helping people to run all Instrumentation test (without `@LargeTest`s).